src/jmcomic/jm_toolkit.py

```
@@ -368,6 +368,25 @@ def limit_text(cls, text: str, limit: int) -> str:
         length = len(text)
         return text if length <= limit else (text[:limit] + f'...({length - limit}')
 
+    @classmethod
+    def get_album_cover_url(cls,
+                            album_id: str,
+                            image_domain: str = None,
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)